AppCenter/AppCenter/Internals/Model/Util/MSUserDefaults.m

```diff
@@ -1,42 +1,84 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#import "MSUserDefaults.h"
 #import "MSAppCenterInternal.h"
 #import "MSLogger.h"
-#import "MSUserDefaults.h"
 
 static NSString *const kMSUserDefaultsTs = @"_ts";
+static NSString *const kMSAppCenterUserDefaultsMigratedKeyFormat = @"%@UserDefaultsMigratedKey";
+static NSString *const kMSUserDefaultsPrefix = @"MSAC";
```

Comment:
Should we add a more descriptive prefix such as mentioned in this PR `MSAppCenter`. Might be more helpful debugging user defaults value for the app dev.